sycl/doc/extensions/Assert/level-zero.md

```diff
@@ -0,0 +1,19 @@
+# Overview
+
+This extension enables detection of assert failure of kernel.
+
+# New enum value
+
+`ze_result_t` enumeration should be augmented with `ZE_RESULT_ABORTED` enum
```

Comment:
should it be ABORTED ? we cannot abort a kernel on GPU - we can only register ASSERT condition and abort on HOST side .

Reply:
It's a typo.